build.sbt

```diff
@@ -12,11 +12,7 @@ inThisBuild(
         "john@degoes.net",
         url("http://degoes.net")
       )
-    ),
-    pgpPassphrase := sys.env.get("PGP_PASSWORD").map(_.toArray),
-    pgpPublicRing := file("/tmp/public.asc"),
-    pgpSecretRing := file("/tmp/secret.asc"),
-    scalaVersion  := Scala3
```

Comment:
Note that the `stdProjectSettings` would still set this one as default. This is probably fine, though different than the rest of the projects. I propose to move those helpers to `BuildHelper`.